docs/Makefile

```diff
@@ -18,15 +17,3 @@ help:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
-
-clean:
-	rm -rf $(BUILDDIR)/*
-	rm -rf api cache gen_modules __pycache
-
-clean-api:
-	rm -rf api
```

Comment:
I don't think this is a defined environment within Sphinx, but we could add this back later if we so choose.